### PR TITLE
Add caloParams_v0_4_cfi using latest 2023 v0 MET PUM LUT

### PR DESCRIPTION
**Pull request by bundocka**
This adds a new caloParams to configure the calo conditions using the latest MET PUM LUT.

### CODE REVIEW

**Comment by aloeliger** [comment]
please test

**Comment by aloeliger** [comment]
please test

**Comment by aloeliger** [comment]
+l1

File: L1Trigger/Configuration/python/customiseSettings.py [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+def L1TSettingsToCaloParams_2023_v0_4(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_4_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_3(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_3_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_2(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_2_cfi")
```
Comment (perrotta) [inline L1Trigger/Configuration/python/customiseSettings.py:14]: Where are all these configs?
I can't see them in L1Trigger/L1TCalorimeter/python

File: L1Trigger/L1TCalorimeter/python/caloParams_2023_v0_4_cfi.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 
+        1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 
+        1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 
+        1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00, 1.00
+    ]),
+
+    # HCal FB LUT
+    layer1HCalFBLUTUpper = cms.vuint32([
+    0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 
+    ]),
```
Comment (perrotta) [inline L1Trigger/L1TCalorimeter/python/caloParams_2023_v0_4_cfi.py:117]:
```suggestion
    layer1HCalFBLUTUpper = [
    0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA ],
```

File: L1Trigger/L1TCalorimeter/python/caloParams_2023_v0_4_cfi.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    layer1HCalFBLUTLower = cms.vuint32([
+    0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 
+    ])
```
Comment (perrotta) [inline L1Trigger/L1TCalorimeter/python/caloParams_2023_v0_4_cfi.py:121]:
```suggestion
    layer1HCalFBLUTLower = [
    0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA, 0xBBBABBBA ]
```

File: L1Trigger/Configuration/python/customiseSettings.py (hunk 1 above)
Comment (aloeliger) [reply L1Trigger/Configuration/python/customiseSettings.py:14]: These are all present in the L1 offline fork. I've been requesting phase 1 L1 developers discontinue use of our generic L1 offline fork to do away with the difficulties of maintaining parallel forks and (un)duplicate information, and it looks like these never made the jump. I can open a PR to @bundocka 's branch to include these.

Comment (aloeliger) [reply L1Trigger/Configuration/python/customiseSettings.py:14]: @bundocka Could you either merge https://github.com/bundocka/cmssw/pull/4 or remove the v0_1 through v0_3 customization options?

**Comment by perrotta** [comment]
please test

**Comment by aloeliger** [comment]
+l1

**Comment by perrotta** [comment]
+1
- The adjustments to the configs suggested [above](https://github.com/cms-sw/cmssw/pull/42225#pullrequestreview-1525048096) can even be applied later on, possibly for all them, i.e. not only the new one added here

**Pull request merged**